core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -354,6 +354,19 @@ public void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, final MountPOption
     });
   }
 
+  @Override
+  public void updateMount(AlluxioURI alluxioPath, final MountPOptions options)
+      throws IOException, AlluxioException {
+    checkUri(alluxioPath);
+    rpc(client -> {
+      MountPOptions mergedOptions = FileSystemOptions.mountDefaults(
+          mFsContext.getPathConf(alluxioPath)).toBuilder().mergeFrom(options).build();
+      client.updateMount(alluxioPath, mergedOptions);
+      LOG.info("UpdateMount on " + alluxioPath.getPath());
```

Comment:
debug level and use `{}` syntax

Reply:
We already log similar information for mount and unmount and I feel it can be useful for understanding the state of the file system. If you think it can be too chatty, I can demote all similar log entries.

Reply:
Actually it seems `mount` is the only other command that logs with info. I will update the level for both.